matRad_sparseBeamletsReaderMSsquare.c

```diff
@@ -0,0 +1,171 @@
+/*
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2019 the matRad development team. 
+% 
```

Comment:
The license of this file needs adjustment as it is copied in large from MIRopt. Potentially we should put it with the other MCsquare code into the submodule and keep in under the Apache license